inference-engine/tools/compile_tool/CMakeLists.txt

```diff
@@ -48,5 +48,5 @@ add_cpplint_target(${TARGET_NAME}_cpplint FOR_TARGETS ${TARGET_NAME})
 # install
 
 install(TARGETS compile_tool
-        RUNTIME DESTINATION ${IE_CPACK_RUNTIME_PATH}
+        RUNTIME DESTINATION deployment_tools/tools/compile_tool
```

Comment:
should not we also add install rules for README files?

Reply:
Added